src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1288,25 +1288,25 @@ class FrontendGenImpl {
     auto input_names = op->getAttrOfType<mlir::ArrayAttr>("input_names");
     auto output_names = op->getAttrOfType<mlir::ArrayAttr>("output_names");
 
-    std::string const sf32 = std::string("f32");
-    std::string const sf64 = std::string("f64");
-    std::string const si32 = std::string("i32");
-    std::string const si64 = std::string("i64");
-    std::string const si16 = std::string("i16");
-    std::string const si8 = std::string("i8");
-    std::string const si1 = std::string("i1");
-    std::string const sui32 = std::string("ui32");
-    std::string const sui64 = std::string("ui64");
-    std::string const sui16 = std::string("ui16");
-    std::string const sui8 = std::string("ui8");
+    std::string const sf32 = std::string(" f32 ");
```

Comment:
Can you offer some insight why adding the spaces matters here, and if its not easier to add them where the string is used?